src/client/js/components/More/More.jsx

```diff
@@ -21,20 +24,23 @@ const buildLinks = pathname => {
   return {
     currentPage,
     prevLinkEnabled: currentPage >= 2,
+    nextLinkEnabled: !(currentCount < PAGE_SIZE),
     prevLink: Number.isInteger(currentPage) && currentPage > 2 ? `/${pathParts[1]}/page/${prevPage}` : `/${pathParts[1]}`,
     nextLink: `/${pathParts[1]}/page/${nextPage}`
   }
 }
 
 const styles = stylesheet({
   more: {
-    padding: `1em 0 1em 0`,
     background: colors.LIGHTEST_GRAY,
     textAlign: 'right',
     gridColumnStart: '1',
     gridColumnEnd: '4',
     gridRowStart: '2',
-    marginRight: '6px'
+    padding: '1em',
+    borderBottom: `6px solid ${colors.NEARFORM_BRAND_MAIN}`,
+    position: 'sticky',
+    top: '70px' // Height of the header TODO: should be shared constant
```

Comment:
Same here

Reply:
this one would require a refactor to improve clarity and separation of concerns on the global styles, theme and layout side which should be a separate issue since it is overarching enough.